app/src/main/java/com/brewfind/ManagePageActivity.kt

```diff
@@ -0,0 +1,11 @@
+package com.brewfind
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+class ManagePageActivity : AppCompatActivity() {
```

Comment:
Created an activity that connects with your manage page layout. This way you should be able to setup button functionality from your main activity that takes you here.
Source: https://dev.to/carolmusyoka/how-to-start-another-activity-on-button-click-using-kotlin-50lk

Reply:
Is this used anywhere?  I don't see any other reference to it.  If it's not used anywhere, it's an incomplete feature.